core/server/web/site/app.js

```diff
@@ -119,7 +119,7 @@ module.exports = function setupSiteApp(options = {}) {
 
     // Members middleware
     // Initializes members specific routes as well as assigns members specific data to the req/res objects
-    siteApp.get('/members/ssr', membersMiddleware.login);
+    siteApp.get('/members/ssr', membersMiddleware.getIdentityToken);
     siteApp.delete('/members/ssr', membersMiddleware.logout);
```

Comment:
If login is being renamed then maybe making logout more explicit: `deleteSession` would also make sense? Not 100% sure about it as it's not critical to know what exactly happens behind logout :man_shrugging:

Reply:
Yeah I can agree with that :relaxed: